src/test/java/uk/gov/hmcts/cmc/claimstore/events/claimantresponse/ClaimantResponseActionHandlerTest.java

```diff
@@ -0,0 +1,64 @@
+package uk.gov.hmcts.cmc.claimstore.events.claimantresponse;
+
+import org.junit.Before;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.mockito.Mock;
+import org.mockito.junit.MockitoJUnitRunner;
+import uk.gov.hmcts.cmc.claimstore.services.notifications.NotificationToDefendantService;
+import uk.gov.hmcts.cmc.claimstore.services.staff.ClaimantRejectOrgPaymentPlanStaffNotificationService;
+import uk.gov.hmcts.cmc.domain.models.Claim;
+import uk.gov.hmcts.cmc.domain.models.sampledata.SampleClaim;
+import uk.gov.hmcts.cmc.domain.models.sampledata.SampleClaimantResponse;
+
+import java.time.LocalDateTime;
+
+import static org.mockito.ArgumentMatchers.eq;
+import static org.mockito.Mockito.times;
+import static org.mockito.Mockito.verify;
+import static uk.gov.hmcts.cmc.claimstore.utils.VerificationModeUtils.once;
+
+@RunWith(MockitoJUnitRunner.class)
+public class ClaimantResponseActionHandlerTest {
+
+    private ClaimantResponseActionsHandler handler;
+
+    @Mock
+    private NotificationToDefendantService notificationService;
+    @Mock
+    private ClaimantRejectOrgPaymentPlanStaffNotificationService claimantRejectOrgPaymentPlanStaffNotificationService;
+
+    @Before
+    public void setUp() {
+        handler = new ClaimantResponseActionsHandler(
+            notificationService,
+            claimantRejectOrgPaymentPlanStaffNotificationService
+        );
+    }
+
+    @Test
+    public void shouldNotifyDefendantOfIntentToProceed() {
+        Claim claim = SampleClaim.builder()
+            .withClaimantRespondedAt(LocalDateTime.now())
+            .withClaimantResponse(SampleClaimantResponse.validRejectionWithDirectionsQuestionnaire())
+            .build();
+        ClaimantResponseEvent event = new ClaimantResponseEvent(claim);
+        handler.notifyDefendantOfIntentToProceed(event);
+
+        verify(notificationService, once())
+            .notifyDefendantOfClaimantResponse(eq(event.getClaim()));
+    }
+
+    @Test
+    public void shouldNotNotifyDefendantOfIntentToProceedWhenClaimantDoesNotIntentToProceed() {
+        Claim claim = SampleClaim.builder()
+            .withClaimantRespondedAt(LocalDateTime.now())
+            .withClaimantResponse(SampleClaimantResponse.validDefaultRejection())
+            .build();
+        ClaimantResponseEvent event = new ClaimantResponseEvent(claim);
+        handler.notifyDefendantOfIntentToProceed(event);
+
+        verify(notificationService, times(0))
+            .notifyDefendantOfClaimantResponse(eq(event.getClaim()));
```

Comment:
`once()` is redundant
`times(0)` should be `never()`
`eq()` isn't needed in this case

Reply:
done. 👍 

`never()` is a wrapper around `times(0)`. 
```
 @CheckReturnValue
    public static VerificationMode never() {
        return times(0);
    }
```